doc/conf.py

```diff
@@ -106,7 +106,7 @@ def __getattr__(cls, name):
 
 # General information about the project.
 project = u'Pyspectral'
-copyright = u'2013-2018, PyTroll'
+copyright = u'2013-2021, Pytroll'
```

Comment:
Depending on how bored you are you could make this use the current year by importing datetime and doing:

```suggestion
copyright = f'2013-{datetime.utcnow():%Y}, Pytroll'
```

Reply:
Ok, yes, I agree. Can we defer this to a more general discussion on how we should handle the headers in all Pytroll code? I rather don't want to change this now in this PR.

Reply:
I'm not sure why it needs to be discussed. Plus this year in this PR is already wrong as it should probably be 2022, right? As for other pytroll code, pyresample and satpy both do the equivalent:

https://github.com/pytroll/pyresample/blob/main/docs/source/conf.py#L67

https://github.com/pytroll/satpy/blob/main/doc/source/conf.py#L116